tests/acceptance/01_vars/02_functions/gettags.cf.expected.json

```diff
@@ -0,0 +1,35 @@
+{
+  "tags1": [
+    "source=promise",
+    "mytag1"
+  ],
+  "tags2": [
+    "source=promise"
+  ],
+  "tags3": [
+    "source=promise",
+    "mytag3"
+  ],
+  "tags4": [
+    "source=promise"
+  ],
+  "tags5": [
+    "source=promise",
+    "mytag5",
+    "mytag51"
+  ],
+  "tags8": [
+    "1",
+    "2"
+  ],
+  "tags9": [
+    "cf_null"
+  ],
+  "tagsa": [
+    "1",
+    "2"
+  ],
+  "tagsb": [
+    "cf_null"
```

Comment:
Hmm these `cf_null` references make me think this will not pass tests. Can you rebase?